examples/access-control-migration/src/models/project.model.ts

```diff
@@ -3,32 +3,32 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Entity, model, property, belongsTo} from '@loopback/repository';
+import {belongsTo, Entity, model, property} from '@loopback/repository';
 import {User} from './user.model';
 
-@model({settings: {strict: false}})
+@model({settings: {strict: false, idInjection: true}})
 export class Project extends Entity {
   @property({
     type: 'number',
-    id: true,
+    id: 1,
     generated: false,
+    updateOnly: true,
```

Comment:
I have a few concerns here but they might not be related to the PR itself:
- Is `settings. idInjection` from model migration? While I was doing task https://github.com/strongloop/loopback-next/pull/4829, I removed `idInjection` from the LB4 docs because it doesn't work when I tested it ( I might test it incorrectly)
- LB3 uses  `id: 1` cause it allows composite ids, and 1 indicates true and also its index. Ideally we should just use `id: true` for imported models.

Reply:
@agnes512 I didn't have it before, it's generated by migration CLI. If it's not correct, I think we'd better fix it from CLI side.

Reply:
True, but I don't know how much effort it needs

Reply:
ah got it after reading the story #4829 , let me remove them.

Reply:
I check what @bajtos did in his PR: https://github.com/strongloop/loopback-next/pull/4737/files#r383816292

And this is what I tried:
1. Define a LB4 model with `idInjection: true`
```@model({
  settings: {
    idInjection: true,
  },
})
export class User extends Entity {
  // @property({   // no id property defined
  //   type: 'number',
  //   id: true,
  //   generated: true,
  // })
  // id?: number;

  @property({
    type: 'string',
  })
  name: string;
...
}
```

2. then run migration-> failed.

I am not hundred percent sure if `idInjection` works in LB4. I am okay to keep it for sake of consistency.

Reply:
@agnes512 many thanks for double checking and verifying with code! 
> I am okay to keep it for sake of consistency.

If it causes failure I'd like to remove in the example, I can investigate a bit (after having PR ready for https://github.com/strongloop/loopback-next/issues/4521) and create issue/PR to fix if needed.